CHANGELOG.md

```diff
@@ -27,6 +27,9 @@
 
 ### Added
 
+- `--no-gc` flag to daemon.
+  [#2721](https://github.com/ChainSafe/forest/issues/2721)
```

Comment:
It'd be great to follow the guidelines above, that is to say, issue first, then a short description.

Reply:
Hmm, the format I use follows what is in the 0.7.0 section. Would u suggest keep them consistent for now and make changes altogether or just change it in this PR?

Reply:
I added the guideline after v.0.7.0 was released. It's too much hassle to rewrite everything from the past, but following the guideline now would be great.